### PR TITLE
Update policies.md

### PR DESCRIPTION
**Pull request by dstewardSP**

### CODE REVIEW

File: scrum/policies.md [hunk 1]
```diff
@@ -206,6 +206,7 @@ There are various formats that a card's Story Points will go under after the com
 
 - `1SP (was 1SP, ...)` - 1SP planned this sprint, 1 left over from last sprint.
 - `2SP + 1 retro` - Additional 2 story points required and updated after release.
```
Comment (MPBeholder) [inline scrum/policies.md]: Shouldn't this be Additional 1 SP required after release, etc?

File: scrum/policies.md [hunk 2]
```diff
@@ -226,6 +227,9 @@ These metrics are then recorded into the "Sprint Summary Card" at the beginning
 Card
 : This refers to a Trello Card. An example card had been created here to understand its parts.
 
+Custom Field
+: When opening a card, there are modules that can be activated for that card. One of them allows for a new field that can contain any information the card owner requires. Usually, this is used to keep track of a card's 'total' Story Points.
```
Comment (MPBeholder) [inline scrum/policies.md]: This is a little generic, is it worth adding an image to help?

**Review by MPBeholder — CHANGES_REQUESTED** [review]

File: scrum/policies.md (hunk 1 above)
Comment (dstewardSP) [reply scrum/policies.md]: I am not sure what you are asking here?

File: scrum/policies.md (hunk 2 above)
Comment (dstewardSP) [reply scrum/policies.md]: I agree now that you say that. I will add one in. Thank you!

File: scrum/policies.md (hunk 1 above)
Comment (MPBeholder) [reply scrum/policies.md]: Discussed w/ Duncan over zoom, resolved

**Pull request merged**